courses/forms.py

```diff
@@ -145,6 +145,12 @@ def program_requirements_schema():
                                     },
                                 },
                             },
+                            "elective_flag": {
+                                "type": "boolean",
+                                "title": "Contains Electives",
+                                "default": False,
+                                "options": {},
```

Comment:
Will this field only be applicable for operators with `MIN_NUMBER_OF`?  If so, it might be nice to have something like:
```
"elective_flag": {
                                "type": "boolean",
                                "title": "Contains Electives",
                                "default": False,
                                "options": {
                                    "dependencies": {
                                        "node_type": ProgramRequirementNodeType.OPERATOR.value,
                                        "operator": ProgramRequirement.Operator.MIN_NUMBER_OF.value,
                                    }
                                },
                            },
```
So the field doesn't exist on every item in the program requirements tree.

Reply:
Did this in https://github.com/mitodl/mitxonline/pull/1736/commits/a483bcf205e7993f10b15f8df0930fc005a269ea but only if `node_type` is operator (see previous note about electives potentially starting with an "all of").